### ISSUE TITLE
Open the DNS query function on port 80

### ISSUE THREAD

**Issue by c2xusnpq6** [AdguardTeam/AdGuardHome#4733]
### Prerequisites

- [y] I am running the latest version
- [y] I checked the documentation and found no answer
- [y] I checked to make sure that this issue has not already been filed

### Problem Description
I am using AWS Lightsail to set up AdGuardHome and then Cloudfront to get HTTPS. Can I use the encrypted DNS feature of AdGuardHome without putting an SSL certificate in AdGuardHome?

### Proposed Solution
Open the DNS query function on port 80?

### Alternatives Considered
Another software?......

### Additional Information
In mainland China and Russia alone, as many as 1.6 billion people are facing serious DNS censorship problems.

But now they can use Cloudflare, Cloudfront or other similar services to bypass censorship.

Because Cloudflare and Cloudfront both offer free credit, and there are many people facing DNS censorship, so I think this feature can help many people, and I believe more and more people will come here to request this feature, that's why I hope you can add this feature. 😁😃

**Comment by c2xusnpq6** [comment]
I hope you guys can give us a detailed tutorial. 😁😅😅

**Comment by fernvenue** [comment]
Yes, of course, please read the [documentation](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#disable-doh-encryption-on-adguard-home).

**Comment by cyb3rz3us** [comment]
> Can I use the encrypted DNS feature of AdGuardHome without putting an SSL certificate in AdGuardHome?

If you mean to service inbound encrypted DNS queries, then No.  AGH requires requires a cert to setup a valid encryption config which is required for AGH to service inbound encrypted queries.  However, AGH can send encrypted queries to upstream providers without a cert installed.

> Open the DNS query function on port 80?

I presume you mean encrypted DNS and if so, then yes but again, this will require a cert.

**Comment by c2xusnpq6** [comment]
![IMG_20220716_211840_424.jpg](https://user-images.githubusercontent.com/75058881/179356612-8d52f330-f9a8-4705-96c1-458b473428e9.jpg)

![IMG_20220716_211849_465.jpg](https://user-images.githubusercontent.com/75058881/179356623-fc0ffde1-0181-44bb-b343-f46b55322bc9.jpg)

I have tried the following but all failed. @cyb3rz3us @fernvenue

```
http://18.177.**.**:443/dns-query
http://18.177.**.**/dns-query
http://18.177.**.**:80/dns-query
```

**Comment by c2xusnpq6** [comment]
Is there a detailed tutorial? thx.

**Comment by c2xusnpq6** [comment]
I just want a httpdns (or httpsdns?) service on port 80, even that it isn't encrypted...

**Comment by cyb3rz3us** [comment]
```
http://18.177.**.**:443/dns-query
http://18.177.**.**/dns-query
http://18.177.**.**:80/dns-query
```

Is this supposed to be a DoH query?   If so, then you will need `https` but even if you had that, it looks like you have the `https_port` set to 0 which is odd.  However, iIf you want a plain DNS query, just use `nslookup`.

As for plain DNS to port 80, I think you can modify this section in the `AdGuardHome.yaml` file:

```
dns:
  bind_hosts:
  - 0.0.0.0
  port: 53
```

But be advised, you may need to change this section at the top of the `AdGuardHome.yaml` file:

```
bind_host: 0.0.0.0
bind_port: 80
```

**Comment by c2xusnpq6** [comment]
> ```
> http://18.177.**.**:443/dns-query
> http://18.177.**.**/dns-query
> http://18.177.**.**:80/dns-query
> ```
> 
> Is this supposed to be a DoH query?   If so, then you will need `https` but even if you had that, it looks like you have the `https_port` set to 0 which is odd.  However, iIf you want a plain DNS query, just use `nslookup`.
> 
> As for plain DNS to port 80, I think you can modify this section in the `AdGuardHome.yaml` file:
> 
> ```
> dns:
>   bind_hosts:
>   - 0.0.0.0
>   port: 53
> ```
> 
> But be advised, you may need to first change this section at the top of the `AdGuardHome.yaml` file:
> 
> ```
> bind_host: 0.0.0.0
> bind_port: 80
> ```

Should I set `https_port` to `80`? Or set it to `443`?...

**Comment by cyb3rz3us** [comment]
> Should I set https_port to 80? Or 443?

Well, If I understand the current situation correctly, DNS-over-HTTPS (DoH) is not working at all so it's probably a good idea to get the default working first and then tweak from there.

With that in mind, I would set the https_port to 443
